docs/user-guide/additional-operator-notes.md

```diff
@@ -4,3 +4,9 @@
 - When deploying CNV on Single Node OpenShift (SNO), [hostpath-provisioner](https://github.com/kubevirt/hostpath-provisioner) (part of the CNV product) storage is automatically opted in and set up to use, to enable persisting VM disks.  
 This is done with the thought in mind that most virtualization use cases require persistence.  
 The hostpath-provisioner is set up to utilize an LSO PV as the backing storage for provisioning dynamic hostPath volumes on.
+
+## Multi-Cluster Engine (MCE)
+
+- When deploying MCE together with a storage operator (ODF or LVM) Infrastructure Operator will be automatically
+enabled. This will require extra disk space availability requirements for the storage operator.
```

Comment:
Maybe a future note of just how much extra disk space availability is needed. Maybe we can just point to ACM docs of disk space required for MCE?

Reply:
I've left them out on purpose because the docs will diverge with the real number (I'd expect the exact space requirement could change), but I can add them if we think it's valuable information

Reply:
That makes sense. I was just imagining customers asking what disk size to use if the validation fails 😆

Reply:
Good point, ODF validation is already stating that, but I can definitely add it to LVM error message